### PR TITLE
MAINT: Standardize searchsorted signature

### PR DESCRIPTION
**Pull request by gfyoung**
"value" is the law of the land.

xref #14645.

Follow-up to #15601.

### CODE REVIEW

File: doc/source/whatsnew/v0.24.0.txt [hunk 1]
```diff
@@ -557,6 +557,7 @@ Deprecations
 - :func:`pandas.read_table` is deprecated. Instead, use :func:`pandas.read_csv` passing ``sep='\t'`` if necessary (:issue:`21948`)
 - :meth:`Series.str.cat` has deprecated using arbitrary list-likes *within* list-likes. A list-like container may still contain
   many ``Series``, ``Index`` or 1-dimensional ``np.ndarray``, or alternatively, only scalar values. (:issue:`21950`)
+- :meth:`FrozenNDArray.searchsorted` has deprecated the ``v`` parameter in favor of ``value`` (:issue:`14645`)
```
Comment (TomAugspurger) [inline doc/source/whatsnew/v0.24.0.txt]: This will cause a warning since FrozenNDArray isn't in our api.rst. Where does the user actually see this method?

**Comment by gfyoung** [comment]
@TomAugspurger : Change was made, and everything is still green.  PTAL.

**Comment by gfyoung** [comment]
@jreback @TomAugspurger : Reverted back to deprecation warning.  If no objections, will merge.

**Comment by jreback** [comment]
ok, pls make sure to add to the deprecations page, otherwise lgtm.

**Pull request merged**